proto/mls/message_contents/group_mutable_metadata.proto

```diff
@@ -17,6 +17,8 @@ message GroupMutableMetadataV1 {
   // The ED25519 private key used to sign commit log entries
   // Must match the first entry in the commit log to be valid
   optional bytes commit_log_signer = 4;
+  // List of users want to leave the group
+  Inboxes pending_remove_list = 5;
```

Comment:
old clients could overwrite this value while adjusting other values, we need to move it to the attributes.